com_migratetojoomla/admin/src/Helper/DownloadHelper.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_migratetojoomla
+ *
+ * @copyright   (C) 2006 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+
+namespace Joomla\Component\MigrateToJoomla\Administrator\Helper;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+
+use Joomla\Component\MigrateToJoomla\Administrator\Helper\HttpHelper;
+use Joomla\Component\MigrateToJoomla\Administrator\Helper\FilesystemHelper;
+use Joomla\Component\MigrateToJoomla\Administrator\Helper\FtpHelper;
+use Joomla\Component\MigrateToJoomla\Administrator\Helper\MainHelper;
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+
+class DownloadHelper
+{
+    public  static $downloadmanager;
+    /**
+     * Method to check connection with respective method
+     * 
+     * @param array form data
+     * @return boolean True on success
+     * 
+     * since
```

Comment:
I would probably use 
@since  1.0
and not @since __DEPLOY_VERSION__
like we usually have in Joomla, as it is a separate project